packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -234,7 +234,12 @@ module.exports = {
                 options: {
                   // @remove-on-eject-begin
```

Comment:
This is removed on eject. Where do we decide what to write on eject? Seems like we'd lose this. Maybe it means I wasn't right about the app default.

Reply:
https://github.com/facebook/create-react-app/pull/5093/files#diff-64d71298908161d0a835f0b72170c307R211